Fixes broken formatting in the docs (#1352)
nucypher/nucypher: docs/source/architecture/contracts.md

## Deployment Procedure

> This section describes how NuCypher contracts are deployed, step-by-step.

Reviewer: ```suggestion
This section provides a step-by-step guide of how NuCypher contracts are deployed.
```

When calculating locked tokens using the `StakingEscrow.getLockedTokens(address, uint16)` method, all stakes that are active during the specified period are summed.

### The Staker Bonds to a Worker ("Ursula")

> The staker must specify a worker who will confirm the activity and sign on behalf of this staker by calling the `StakingEscrow.setWorker(address)` method.

Reviewer 2: Should it be backquotes or double-backquotes?

Author: Single quotes in Markdown, double quotes in RST.

Reviewer 2: Ah true, that's markdown...

When calculating locked tokens using the `StakingEscrow.getLockedTokens(address, uint16)` method, all stakes that are active during the specified period are summed.

### The Staker Bonds to a Worker ("Ursula")
The staker must specify a worker who will confirm the activity and sign on behalf of this staker by calling the `StakingEscrow.setWorker(address)` method.

> Changing a worker is allowed no more than 1 time in `StakingEscrow.minWorkerPeriods()`.

Reviewer: ```suggestion
Changing a worker is allowed no more than once within `StakingEscrow.minWorkerPeriods()`.
```

Some users will have locked but not staked tokens.
In that case, an instance of the `UserEscrow` contract will hold their tokens (method `UserEscrow.initialDeposit(uint256, uint256)`).
All tokens will be unlocked after a specified time and the user can retrieve them using the `UserEscrow.withdraw(uint256)` method.

> When the user wants to become a staker - she uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.

Reviewer 2: !

Author: 👩‍🚀

Some users will have locked but not staked tokens.
In that case, an instance of the `UserEscrow` contract will hold their tokens (method `UserEscrow.initialDeposit(uint256, uint256)`).
All tokens will be unlocked after a specified time and the user can retrieve them using the `UserEscrow.withdraw(uint256)` method.

> When the user wants to become a staker - she uses the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.

Reviewer 3: :wink: 

```suggestion
When the user wants to become a staker - they use the `UserEscrow` contract as a proxy for the `StakingEscrow` and `PolicyManager` contracts.
```
